README.md

```diff
@@ -1,4 +1,7 @@
 # lk2nd
+
+![](FOSDEM22_msm8916_lk2nd.png)
```

Comment:
```suggestion
![](FOSDEM22_msm8916_lk2nd.jpg)
```

:)

but tbh I'd consider if we should hide that file somewhere into `.github/` or some dedicated `.assets/` along with the license file, so it doesn't catch the eye too much. Also can probably drop "msm8916" since there are a couple of 8226s on the photo too.